### DISCUSSION TITLE
System topics ?

### DISCUSSION THREAD

**Discussion by RCdotNet** [dotnet/MQTTnet#1313] [Q&A]
Hi,
I'm new here and I have a little question about MQTTnet. Is there any System topics implemented? 
I was not able to find any reference to that.
Thanks in advance!

**Comment by SeppPenner** [comment]
System topics are not yet supported neither is the support planned yet. Check out https://github.com/dotnet/MQTTnet/issues/1258:

> No it is currently not supported and not planned yet. But you can build this on your own via sending a reply on certain topics.
